packages/pool/contracts/StateUtils.sol

```diff
@@ -344,6 +365,26 @@ contract StateUtils is IStateUtils {
             );
     }
 
```

Comment:
might be nice to have a function that will just `push` one new `votingApp` to the array, rather than update the entire thing